### DISCUSSION TITLE
Allow only admins to revert changes to document

### DISCUSSION THREAD

**Discussion by saikat** [outline/outline#7109] [Ideas]
I'm interested in using Outline for a semi-public knowledgebase where many people would be able to contribute (something like a Wikipedia use case). In this scenario, it would be really helpful to have a distinction between people who can edit a document and people who can revert other people's edits. I think the simplest approach to this would be to only allow admins to have the ability to change document history, though I know that would probably not work for a lot of people using the tool as-is currently. The more complex approach might be to build out more fine grained ACL (maybe similar to Wiki.js).

**Comment by tommoor** [comment]
A revert is just another type of edit, so I think this draws a distinction that doesn't really exist – as I could easily copy and paste the content from history if revert was disabled but I retained edit access.
